src/common/fam_internal_exception.h

```diff
@@ -188,6 +193,7 @@ inline enum Fam_Error convert_to_famerror(enum Internal_Error serverErr) {
     case ITEM_REGISTRATION_FAILED:
     case ITEM_DEREGISTRATION_FAILED:
     case DATAITEM_SIZE_TOO_LONG:
+    case BACKUP_SIZE_TOO_LONG:
```

Comment:
Should we use "LARGE" in place of "LONG" while using it with data item/backup "size" ?
minor comment, you can ignore this.